drivers/scheduler/k8s/k8s.go

```diff
@@ -2929,32 +2962,104 @@ func (k *K8s) destroyBackupObjects(
 	return nil
 }
 
-// getExpectedAutopilotSizeForPvc calculates exected size of PVC based on autopilot rules and workload
-func (k *K8s) getExpectedAutopilotSizeForPvc(pvc *v1.PersistentVolumeClaim, apRules *apapi.AutopilotRuleList, wSize uint64) (uint64, error) {
-	var (
-		err             error
-		expectedPVCSize uint64
-	)
+// GetExpectedAutopilotVolumeExpand calculates expected size of PVC based on autopilot rule and workload
+func (k *K8s) GetExpectedAutopilotVolumeExpand(pvc *v1.PersistentVolumeClaim, apRule apapi.AutopilotRule, wSize uint64) (uint64, int, error) {
```

Comment:
this does not need to be a public method. Only current package is using it.

Reply:
The name `GetExpectedAutopilotVolumeExpand`doesn't sound grammatically correct. Suggest renaming to  `estimatePVCExpansion`